app/assets/stylesheets/_reports.scss

```diff
@@ -2,10 +2,13 @@
   .container {
     min-height: 100%;
   }
+  .tabs_container{
```

Comment:
Selector `tabs_container` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space